frontend/app/views/testing/testUsers.scala.html

```diff
@@ -36,32 +36,30 @@
-                            (function() {
-                                if (!document.getElementById('register_field_username')) {
-                                    alert('Element with id register_field_username not found. Are you on the right page?');
-                                    return;
-                                }
-                                var username = prompt('Test User key (first name, last name, username and password will be set to this value):');
-                                if (!username) return;
-                                ['firstname', 'lastname', 'username', 'password', 'email'].forEach(function(field) {
-                                    document.getElementById('register_field_'+field).value = field === 'email' ? username + '@@gu.com' : username;
-                                });
-                                document.querySelector('#register_submit').click();
-                            })();
-                        ">
-                                @if(Config.stageDev){Dev } New Identity Signup</a>.
-                            Clicking this on the <em>new</em> identity sign-up page will prompt you for the Test User key, then fill out fields appropriately and submit the form. Unless you're a <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=866522">Firefox user</a>.
-                        </li>
-                        <li>
-                            <a href="javascript:
-                            (function() {
-                                var s = document.createElement('script');
-                                s.src = '@Asset.bookmarklet("identity-signup.js")';
-                                s.type = 'text/javascript';
-                                document.head.appendChild(s);
-                            })();">
-                                @if(Config.stageDev){Dev }Identity Signup</a>.
-                            Clicking this on the identity sign-up page will prompt you for the Test User key, then fill out fields appropriately and submit the form.
+                                (function() {
+                                    var fieldPrefix = 'register_field_';
+                                    function idFor(field) { return fieldPrefix+field; }
+
+                                    if (!document.getElementById(idFor('email'))) {
+                                        alert('Element with id '+idFor('email')+' not found. Are you on the right page?');
+                                        return;
+                                    }
+                                    var testUserToken = prompt('Test User token:');
+                                    if (!testUserToken) return;
+
+                                    ['firstname', 'lastname', 'password', 'email'].forEach(function(field) {
+                                        var element = document.getElementById(idFor(field));
+                                        element.focus();
```

Comment:
I feel that call this functions to simulate the browser maybe is not the best approach. Nevertheless, this code should be refactored in the future and all this chunk of code should disappear.